client/my-sites/stats/stats-module/all-time-nav.jsx

```diff
@@ -87,6 +82,9 @@ export const StatsModuleSummaryLinks = ( props ) => {
 					key={ 'navTabsDropdown-' + index }
 					path={ i.path }
 					selected={ i.value === selected.value }
+					onSelect={ () => {
```

Comment:
```suggestion
					onClick={ () => {
```

I noticed the `onSelect` event on the item is not working since we pass [children to SelectDropdown](https://github.com/Automattic/wp-calypso/blob/update/stats-75742-remove-ff-horizontal-bars/client/components/select-dropdown/index.jsx#L152), and [it triggers the `onClick` event](https://github.com/Automattic/wp-calypso/blob/update/stats-75742-remove-ff-horizontal-bars/client/components/select-dropdown/index.jsx#L164-L167) rather than `onSelect` event passed to its children.